drivers/mlx90393/include/mlx90393_constants.h

```diff
@@ -0,0 +1,118 @@
+                                                 connectivity test */
+#define MLX90393_REG_REF_TEMP       0x24    /**< Reference temperature */
+
+/**
+ * @brief   Configuration parameter bit masks
+ */
+#define MLX90393_MASK_BDR           0x003F  /**< burst data rate */
+#define MLX90393_MASK_WOC_DIFF      0x1000  /**< wake up on change mode (relative or absolute) */
+#define MLX90393_MASK_TCMP_EN       0x0400  /**< enable temperature compensation */
+#define MLX90393_MASK_GAIN_SEL      0x0070  /**< analog chain gain */
+#define MLX90393_MASK_RES_XYZ       0x07E0  /**< xyz resolution */
+#define MLX90393_MASK_COMM_MODE     0x6000  /**< set allowed bus communication (I2C or SPI) */
+#define MLX90393_MASK_OSR           0x0003  /**< oversampling ratio magnetic sensor */
+#define MLX90393_MASK_OSR2          0x1800  /**< oversampling ratio temperature */
+#define MLX90393_MASK_DIG_FILT      0x001C  /**< digital filter magnetic sensor */
+
+/**
+ * @brief   Number of left shifts in configuration registers
+ */
+#define MLX90393_SHIFT_OSR2         11      /**< oversampling ratio temperature */
+#define MLX90393_SHIFT_DIG_FILT     2       /**< digital filter magnetic sensor */
+#define MLX90393_SHIFT_GAIN         4       /**< analog chain gain */
+#define MLX90393_SHIFT_RES_X        5       /**< x resolution */
+#define MLX90393_SHIFT_RES_Y        7       /**< y resolution */
+#define MLX90393_SHIFT_RES_Z        9       /**< z resolution */
+#define MLX90393_SHIFT_WOC_MODE     12      /**< wake-up on change mode */
+
+/**
+ * @brief   Command Set
+ */
+#define MLX90393_COMMAND_SB         0x1F    /**< start burst mode */
+#define MLX90393_COMMAND_SW         0x2F    /**< start wake up on change mode */
+#define MLX90393_COMMAND_SM         0x3F    /**< start single measurement mode */
+#define MLX90393_COMMAND_RM         0x4F    /**< read measurement */
+#define MLX90393_COMMAND_RR         0x50    /**< read register */
+#define MLX90393_COMMAND_WR         0x60    /**< write register */
+#define MLX90393_COMMAND_EX         0x80    /**< exit mode */
+#define MLX90393_COMMAND_HR         0xD0    /**< memory recall */
+#define MLX90393_COMMAND_HS         0xE0    /**< memory store */
+#define MLX90393_COMMAND_RT         0xF0    /**< reset */
+
+/**
+ * @brief   Status byte bit map
+ */
+#define MLX90393_STATUS_RESET       0x04    /**< reset bit */
+#define MLX90393_STATUS_ERROR       0x10    /**< error bit */
+
+/**
+ * @brief   Timeout durations in ms
+ */
+#define MLX90393_COMMAND_EX_TIMEOUT     1    /**< Timeout after exit command */
+#define MLX90393_COMMAND_RT_TIMEOUT     2    /**< Timeout after reset command */
+
+/**
+ * @brief   Temperature conversion constants
+ */
+#define MLX90393_TEMP_OFFSET        3500    /**< Temperature offset in centi celsius */
+#define MLX90393_TEMP_RESOLUTION    452     /**< Temperature sensor resolution
+                                            * (45.2 * 10 for avoiding float values) */
```

Comment:
Ich denke, so wäre es besser als die misaligned Variante
```suggestion
#define MLX90393_TEMP_RESOLUTION    452     /**< Temperature sensor resolution
                                                 (45.2 * 10 for avoiding float values) */
```